examples/examples/z_sub_thr.rs

```diff
@@ -141,6 +142,10 @@ fn parse_args() -> (Config, usize, usize) {
     if args.is_present("no-multicast-scouting") {
         config.scouting.multicast.set_enabled(Some(false)).unwrap();
     }
+    #[cfg(feature = "shared-memory")]
+    if args.is_present("enable-shm") {
+        config.transport.shared_memory.set_enabled(true).unwrap();
+    }
```

Comment:
Add a check on `#[cfg(not(feature = "shared-memory"))]` and `if args.is_present("enable-shm")` saying that SHM can not be enabled because Zenoh has been compiled without `shared-memory` feature.